pom.xml

```diff
@@ -89,37 +90,54 @@
         </pluginManagement>
         <plugins>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
+                <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <!-- jenkins-test-harness < 1.545 doesn't support concurrent tests. -->
```

Comment:
I would suggest bumping it to the version in Parent POM in order to pick more injected tests